src/qt/walletview.cpp

```diff
@@ -95,13 +95,13 @@ void WalletView::setBitcoinGUI(BitcoinGUI *gui)
         connect(this, SIGNAL(message(QString,QString,unsigned int)), gui, SLOT(message(QString,QString,unsigned int)));
 
         // Pass through encryption status changed signals
-        connect(this, SIGNAL(encryptionStatusChanged(int)), gui, SLOT(setEncryptionStatus(int)));
+        connect(this, SIGNAL(encryptionStatusChanged(int)), gui, SLOT(updateWalletStatus()));
```

Comment:
In commit "Qt: Ensure UI updates only come from the currently selected walletView"

This is silently dropping the `int status` value passed to `encryptionStatusChanged`, and the `int hdEnabled` value passed to `hdEnabledStatusChanged.` Assuming this is intended, you should clean up after this change by deleting the unused parameters from `encryptionStatusChanged` and `hdEnabledStatusChanged` declarations and calls.